docs/docs/statistics/power.mdx

```diff
@@ -0,0 +1,211 @@
+
+## How do I interpret power analysis results?
+
+In this section we run through an example power analysis.
+
+After starting power analysis, you will need to select metrics and enter estimated users per week. In our example, we choose a binomial metric (Retention - [1, 14) Days) and a revenue metric (Purchases - Total Revenue (72 hour window)). We refer to these metrics as "retention" and "revenue" respectively going forward. We estimate that 500,000 new users per week visit our website.
+
+![metric and users input](/images/statistics/power_1.png)
+
+You then input your effect sizes, which are the best guesses for your expected metric improvements due to your feature. We provide guidance for effect size selection [here](#how-should-i-pick-my-effect-size). For our feature we anticpate a 2% improvement in retention. For retention, the conversion rate is 10%. This 10% number should come from an offline query that measures your conversion rate on your experimental population. We expect a 1% improvement in revenue, which has a mean of $0.10 and a standard deviation of $0.22 (as with conversion rate, the mean and standard deviation come from an offline query that you run on your population). We then submit our data.
+
+![metric and users input](/images/statistics/power_2.png)
+
+Now we can see the results! At the top of the page is a box called Analysis Settings. The summary here says, "Run experiment for 4 weeks to achieve 80% power for all metrics." This is the most important piece of information from power analysis. If running your experiment for 4 weeks is compatible with your business constraints, costs, and rollout timeframe, then you do not need to dive into the numbers below this statement. If you want to rerun power results with number of variations greater than 2, then click "# of Variations" and then click "Update". If you want to toggle on or off "Sequential Testing", then press the "Edit" button and select the appropriate option. Enabling sequential testing reduces power.
+
+![metric and users input](/images/statistics/power_3.png)
+
+Below Analysis Settings is Calculated Sample Size and Runtime, which contains the number of weeks (or equivalently the number of users) needed to run your experiment to achieve 80% power by metric. Clicking on a row in the table causes the information in the box to the right to change. We expect 80% power for retention if we run the experiment for 2 weeks. For revenue, we need to run the experiment for 4 weeks to achieve at least 80% power.
+
+Calculated Sample Size and Runtime is Power Over Time, which contains power estimates by metric. The columns in Power Over Time correspond to different weeks. For example, in the first week power for retention is 65%. The highlighted column in week 4 is the first week where at least 80% power is achieved for all metrics. As expected, power increases over time, as new users are added to the experiment.
```

Comment:
Typo?

Reply:
it's not a typo, but it is hard to read.  I added some quotes to help, thanks.